main.tf

```diff
@@ -281,3 +281,5 @@ resource "aws_kms_grant" "this" {
   grant_creation_tokens = try(each.value.grant_creation_tokens, null)
   retire_on_delete      = try(each.value.retire_on_delete, null)
 }
+
```

Comment:
please remove these extra lines for the CI checks to pass

Reply:
@bryantbiggs thanks for the information. I has remove the extra lines. Best regards